scripts/partition_manager.py

```diff
@@ -1383,6 +1407,53 @@ def test():
     expect_addr_size(td, 'EMPTY_1', 10400, 600)
     expect_addr_size(td, 'with_alignment_2', 11000, 100)
 
+    # Test 'align_next'
```

Comment:
Could you write one sentence about how this is tested? "'second' will ensure that 'third' is aligned on 4000 and 'third' will ensure that 'fourth' is aligned on 6000' " or something like that.